infrastructure/main.tf

```diff
@@ -103,7 +104,7 @@ module "sma-base" {
     VECTOR_SECRET_NAME    = var.vector_secret_name,
     ASSUME_ROLE_READ_ARN  = var.assume_role_read_arn,
     ASSUME_ROLE_WRITE_ARN = var.assume_role_write_arn,
-    SM2A_BASE_URL         = module.sma-base.airflow_url,
+    SM2A_BASE_URL         = "https://${lower(var.subdomain)}.${var.domain_name}",
```

Comment:
These changes look good but I vaguely recall modifying URLs in the past to better cooperate with veda-routes and the ENV Variables stored for veda-deploy. @botanical do you recall any reason why we wouldn't want to manage the base url this way? I may be remembering an unrelated subdomain+domain config issue.

Reply:
I don't think this change will impact veda-routes or the env vars stored for veda deploy. Maybe you're remembering this conversation https://developmentseed.slack.com/archives/C045K5LAFA9/p1749230112902339 ?